phpBB/styles/prosilver/template/ucp_register.html

```diff
@@ -57,10 +57,12 @@ <h2>{SITENAME} - {L_REGISTRATION}</h2>
 	<hr />
 
 	<!-- EVENT ucp_register_options_before -->
+	{% if S_LANG_OPTIONS %}
 	<dl>
 		<dt><label for="lang">{L_LANGUAGE}{L_COLON}</label></dt>
 		<dd><select name="lang" id="lang" onchange="change_language(this.value); return false;" tabindex="6" title="{L_LANGUAGE}">{S_LANG_OPTIONS}</select></dd>
 	</dl>
+	{% endif %}
```

Comment:
```twig
	{% if S_LANG_OPTIONS is not empty %}
	<dl>
		<dt><label for="lang">{{ lang('LANGUAGE') ~ lang('COLON') }}</label></dt>
		<dd><select name="lang" id="lang" onchange="change_language(this.value); return false;" tabindex="6" title="{{ lang('LANGUAGE') }}">{{ S_LANG_OPTIONS }}</select></dd>
	</dl>
	{% endif %}
```

Reply:
`{% if S_LANG_OPTIONS %}` should work the same way.

Reply:
You modified `S_LANG_OPTIONS ` that's why I think the whole block can be covered, if you so wish.